packages/@ember/runloop/index.ts

```diff
@@ -99,8 +128,20 @@ export const _backburner = new Backburner(_queues, {
   @return {Object} return value from invoking the passed function.
   @public
 */
-export function run() {
-  return _backburner.run(...arguments);
+export function run<F extends (...args: any[]) => any>(method: F, ...args: Parameters<F>): void;
```

Comment:
Given the repeated use of `(...args: any[]) => any` I'd suggest extracting a type alias:

```ts
type AnyFn = (...args: any[]) => any;
```

Then you can use that throughout, and it'll be both briefer and clearer (and be a single place to tell ESLint not to yell at us about `any`).

Reply:
Yeah, makes sense.

Reply:
I think we should do this in its own PR.

Reply:
Agreed re: doing it in its own PR.